lib/main.dart

```diff
@@ -50,7 +25,7 @@ void main() async {
 Future<void> _initFire() async {
   WidgetsFlutterBinding.ensureInitialized();
   await FlutterConfig.loadEnvVariables();
-  await Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
+  await Firebase.initializeApp();
```

Comment:
I think you should revert here. We need to check the current platform.

Related: https://stackoverflow.com/questions/70232931/firebaseoptions-cannot-be-null-when-creating-the-default-app
<img width="689" alt="Screenshot 2022-11-12 at 2 34 14 PM" src="https://user-images.githubusercontent.com/27461460/201458989-30522379-bb3f-4d8f-bb23-9dcfc9e6ef34.png">